### PR TITLE
integrate font assets

### LINKED ISSUE #45: integrate-font-assets
Integrate font assets into project

### PR DESCRIPTION
**Pull request by Remi-dee**
This PR resolves #45  
Created public/assets/font directory,
Added font files to created directory,
Integrated font files at app level

### CODE REVIEW

**Review by annonymousauthority — CHANGES_REQUESTED** [review]

File: src/app/layout.js [hunk 1]
```diff
@@ -11,7 +44,9 @@ export const metadata = {
 export default function RootLayout({ children }) {
   return (
     <html lang="en">
-      <body className={inter.className}>{children}</body>
+      <body className={`${inter.variable}  ${nohemi.variable}`}>
```
[the remark this answers is not in the document]
Comment (Remi-dee) [reply src/app/layout.js]: Done

File: src/data/localFont.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import localFont from "next/font/local";
+
+const nohemi = localFont({
+  src: [
+    {
+      path: "../../public/assets/font/Nohemi-Black-BF6438cc565e67b.woff",
```
Comment (annonymousauthority) [inline src/data/localFont.js:6]: Using `@/public/assets` feels shorter than `../../` could you try that?

File: tailwind.config.js [hunk 3]
```diff
@@ -13,6 +13,10 @@ module.exports = {
         "gradient-conic":
           "conic-gradient(from 180deg at 50% 50%, var(--tw-gradient-stops))",
       },
+      fontFamily: {
+        sans: ["var(--font-inter)"],
```
Comment (annonymousauthority) [inline tailwind.config.js:17]: Take out inter completely from the project. we're currently using only `nohemi`

**Review by annonymousauthority — CHANGES_REQUESTED** [review]

File: src/data/localFont.js (hunk 2 above)
Comment (Remi-dee) [reply src/data/localFont.js:6]: Yes i can, on it.

File: tailwind.config.js (hunk 3 above)
Comment (Remi-dee) [reply tailwind.config.js:17]: done

Comment (annonymousauthority) [inline tailwind.config.js:17]: Remove this

**Review by annonymousauthority — APPROVED** [review]

File: tailwind.config.js (hunk 3 above)
Comment (annonymousauthority) [reply tailwind.config.js:17]: I'll merge after you make that commit

Comment (Remi-dee) [reply tailwind.config.js:17]: My mind totally skipped that😩, on it

Comment (Remi-dee) [reply tailwind.config.js]: Done

**Review by annonymousauthority — APPROVED** [review]
LGTM

**Pull request merged**